libs/internal/include/launchdarkly/data_model/flag.hpp

```diff
@@ -16,8 +16,10 @@
 namespace launchdarkly::data_model {
 
 struct Flag {
-    using Variation = std::uint64_t;
-    using Weight = std::uint64_t;
+    using Variation = std::int64_t;
+    using Weight = std::int64_t;
+    using FlagVersion = std::uint64_t;
```

Comment:
Can't fall this `Version` because it conflicts with the `Version()` method. Hmm.